src/utils/sqs.ts

```diff
@@ -0,0 +1,32 @@
+import { SqsService } from "./types";
+import {
+  SendMessageCommand, SendMessageCommandOutput,
+  SendMessageRequest,
+  SQSClient,
+} from "@aws-sdk/client-sqs";
+import { logger } from "./logger";
+
+export function sqsService(): SqsService {
+  const send = async function (messageBody: string): Promise<any> {
+    const { AWS_REGION } = process.env;
+    const { AUDIT_QUEUE_URL } = process.env;
+    const client = new SQSClient({ region: AWS_REGION });
+    const message: SendMessageRequest = {
+      QueueUrl: AUDIT_QUEUE_URL,
+      MessageBody: messageBody,
+    };
+
+    if (AUDIT_QUEUE_URL != null) {
```

Comment:
If you exit early we can reduce the nesting e.g.
`if (!AUDIT_QUEUE_URL) {
      logger.error(`Environment missing value for AUDIT_QUEUE_URL, cannot send ${messageBody}.`);
      return;
    }`